### PR TITLE
Static import semconv constants in tests

### PR DESCRIPTION
**Pull request by trask**
I was looking into scraping the code to verify that all inlined constants are tested (via non-inlined constants):

https://github.com/open-telemetry/opentelemetry-java-contrib/pull/1604#issuecomment-2552433246

and this normalizing will help my grepping efforts

tracking my latest script:

```
#!/bin/bash -e

for file in $(find instrumentation -name '*Test.java'); do
  echo "Processing $file"

  # stable semconv

  negative_lookbehind='(?<!import static io.opentelemetry.semconv.)'

  for class in "ClientAttributes" "ErrorAttributes" "ExceptionAttributes" "HttpAttributes" "JvmAttributes" "NetworkAttributes" "OtelAttributes" "ServerAttributes" "ServiceAttributes" "TelemetryAttributes" "UrlAttributes" "UserAgentAttributes"; do
    for attr in $(perl -ne "print \"\$1\n\" if /$negative_lookbehind$class\.([A-Z][A-Z][A-Z_]*)/" $file | sort -u); do
      perl -i -pe "s/$negative_lookbehind$class\.$attr/$attr/" $file
      sed -i "0,/^import/{s/^import/import static io.opentelemetry.semconv.$class.$attr;\nimport/}" $file
    done
  done

  # incubating semconv

  negative_lookbehind='(?<!import static io.opentelemetry.semconv.incubating.)'

  for class in "AwsIncubatingAttributes" "DbIncubatingAttributes" "HttpIncubatingAttributes" "MessagingIncubatingAttributes"; do
    for attr in $(perl -ne "print \"\$1\n\" if /$negative_lookbehind$class\.([A-Z][A-Z][A-Z_]*)/" $file | sort -u); do
      perl -i -pe "s/$negative_lookbehind$class\.$attr/$attr/" $file
      sed -i "0,/^import/{s/^import/import static io.opentelemetry.semconv.incubating.$class.$attr;\nimport/}" $file
    done
  done
done

./gradlew spotlessApply
```

### CODE REVIEW

**Comment by zeitlinger** [comment]
have you tried using this approach? 

https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12009/files#diff-b36b9b7d4939f00472feba2fd8ee942cec0591677a9b7a8220b6770ba1cd2fadR53-R64

**Comment by trask** [comment]
> have you tried using this approach?
> 
> https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/12009/files#diff-b36b9b7d4939f00472feba2fd8ee942cec0591677a9b7a8220b6770ba1cd2fadR53-R64

I'm not quite ready to enforce it, maybe after we finish migrating all of the groovy tests

File: instrumentation/hibernate/hibernate-6.0/spring-testing/src/test/java/spring/jpa/SpringJpaTest.java [hunk 1]
```diff
@@ -51,7 +52,7 @@ static void tearDown() {
     }
   }
 
-  @SuppressWarnings("deprecation") // DbIncubatingAttributes.DB_NAME has been deprecated
+  @SuppressWarnings("deprecation") // DB_NAME has been deprecated
```
Comment (laurit) [inline instrumentation/hibernate/hibernate-6.0/spring-testing/src/test/java/spring/jpa/SpringJpaTest.java:55]: `DB_NAME` isn't the only deprecate attribute, maybe reword the comment?

**Review by laurit — APPROVED** [review]
